From: Fabien Givors <f+debian@chezlefab.net>
Date: Mon, 13 Oct 2014 19:26:11 +0200
Subject: make clean now revert to pre-configure state
-
Author: Fabien Givors <f+debian@chezlefab.net>
Forwarded: yes
Last-Update: 2014-10-13


  * make clean remove makefile.mk file
  * make clean restore default PREFIX in config.h
---
 Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile b/Makefile
index a241804..e8924fd 100644
--- a/Makefile
+++ b/Makefile
@@ -48,6 +48,9 @@ mrproper: clean
 
 clean:
 	rm -f obsession-exit obsession-logout xdg-autostart *.o $(I18N_MO)
+	sed -i 's#define PREFIX.*#define PREFIX              "/usr/local"#' config.h
+
+	rm -f makefile.mk
 
 configure:
 	sed -i 's#define PREFIX.*#define PREFIX "$(PREFIX)"#' config.h
